src/main/webapp/js/common.js

```diff
@@ -775,7 +775,7 @@ function highlightSearchResult(searchKeyId, sectionToHighlight) {
     var searchKey = $(searchKeyId).val();
     // trim symbols around every word in the string
     var symbolTrimmedSearchKey = [];
-    $.each(searchKey.split(/["'.-]/), function() {
+    $.each(searchKey.split(/[ "'.-]/), function() {
         symbolTrimmedSearchKey.push($.trim(this));
```

Comment:
Maybe rename this variable (and change the comment above) as this no longer applies to symbols only.